### PR TITLE
docs: minor improvements

### PR DESCRIPTION
**Pull request by RisenID**

### CODE REVIEW

File: docs/expert/build.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ The following dependencies must be installed:
 - [Node.js](https://nodejs.org/) (installing via [nvm](https://github.com/nvm-sh/nvm) is suggested)
 - [protobuf](https://github.com/protocolbuffers/protobuf)
 - [Python 3](https://www.python.org/) (or newer)
-- [Java 11](https://www.java.com/) (or newer)
+- [Java 11](https://www.java.com/)
```
Comment (salvogiangri) [inline docs/expert/build.md]: Why removing "or newer"? I'm currently using Java 17 and it works just fine.

File: docs/guide/bootloader.md [hunk 2]
```diff
@@ -12,10 +12,10 @@ nav_order: 1
   <img loading="lazy" src="/assets/images/bl-unlock.png" width="30%"/>
 </p>
 
-Unlocking your bootloader is a fundamental step when customizing your device, as it allows you to flash and install unofficial images in it. Please note this procedure **will completely wipe your data**, so make a backup before proceeding.
+Unlocking your bootloader is the first step when installing a custom rom your device, as it allows you to flash and install unofficial images on it. Please note this procedure **will completely wipe your data**, so make a backup before proceeding.
```
Comment (salvogiangri) [inline docs/guide/bootloader.md]: Typo: "installing a custom rom your device". I also didn't specify the custom ROM use case as that's not the only reason to unlock the bootloader

File: docs/guide/download.md [hunk 3]
```diff
@@ -66,4 +65,4 @@ If you encounter any error/issue, please refer to the [Troubleshooting]({% link
 
 ## Updating the ROM
 {: .pb-2 }
-To update the ROM, you can simply follow the same exact steps you've followed for the first time install, expection made for the data wipe which is not necessary anymore.
+To update the ROM, you can simply follow the same exact steps you've followed for the first time install, except for for the data wipe which is not necessary anymore.
```
Comment (salvogiangri) [inline docs/guide/download.md]: Typo: "except for for"

File: docs/guide/recovery.md [hunk 4]
```diff
@@ -9,7 +9,7 @@ nav_order: 2
 {: .pb-2 }
 
 {: .important }
-> This guide is specific to devices in which the install method is marked as **"Custom recovery"**.
+> This guide is specific to the devices in which the install method is marked as **"Custom recovery"**.
```
Comment (salvogiangri) [inline docs/guide/recovery.md:12]: This should also be matching in:
https://github.com/BlackMesa123/UN1CA/blob/ea95478e6f0d093a98e6fea51cbdda3fe721a40e/docs/guide/download.md?plain=1#L12

**Review by salvogiangri — CHANGES_REQUESTED** [review]

File: docs/expert/build.md (hunk 1 above)
Comment (RisenID) [reply docs/expert/build.md]: Some of the tools dont compile without java 11

File: docs/guide/bootloader.md (hunk 2 above)
Comment (RisenID) [reply docs/guide/bootloader.md]: Yeah but imo, the older one which said "fundamental step when customising your phone" isnt exactly true

File: docs/expert/build.md (hunk 1 above)
Comment (salvogiangri) [reply docs/expert/build.md]: This has been addressed in 4731196f36bfcd3b237dc55af34579da35f8e0e9 by removing smali in the project. @iDrinkCoffee-TG fixed this in 8fcf3320e6d4a373d5d8ea9c01b31bc8363930e4 but I'm waiting for you to confirm it is okay

File: docs/guide/bootloader.md (hunk 2 above)
Comment (salvogiangri) [reply docs/guide/bootloader.md]: In c9f0856c08de81eefdf0b731b75b20eb0eacf5bc, what about "the first step to follow to customize your device"?

File: docs/guide/download.md (hunk 3 above)
Comment (salvogiangri) [reply docs/guide/download.md]: Addressed in dc0db2711f64c528b0b4b941d3f9866ed57bbace.

File: docs/guide/recovery.md (hunk 4 above)
Comment (salvogiangri) [reply docs/guide/recovery.md:12]: Addressed in 1cf689150b92a7a296f54b6991cc8bf8b51f2cda.

File: docs/guide/bootloader.md (hunk 2 above)
Comment (RisenID) [reply docs/guide/bootloader.md]: customising the device could also just be changing the launcher/modding using something like the goodlock suite, which is why im hesitant to use that word

File: docs/expert/build.md (hunk 1 above)
Comment (RisenID) [reply docs/expert/build.md]: Will test on my end once again

File: docs/guide/bootloader.md (hunk 2 above)
Comment (salvogiangri) [reply docs/guide/bootloader.md]: You're right, maybe "installing a custom ROM or simply rooting your device" fits better?

Comment (RisenID) [reply docs/guide/bootloader.md]: yeah sounds good, made the change, +1 to go ahead and merge this from my side.

File: docs/expert/build.md (hunk 1 above)
Comment (RisenID) [reply docs/expert/build.md]: marking as resolved, since this works fine now

**Review by salvogiangri — APPROVED** [review]

**Pull request merged**